content/test/unit/service/Service.spec.ts

```diff
@@ -45,7 +46,7 @@ describe('Service', function () {
   // let service: ServiceImpl
 
   it('starts the variables', async () => {
-    randomFileHash = await Hashing.calculateBufferHash(randomFile)
+    randomFileHash = await hashV1(randomFile)
```

Comment:
shouldn't this be `hashV0`?

Reply:
New default hashing is V1, fails otherwise